src/main/include/subsystems/WristSubsystem.h

```diff
@@ -0,0 +1,126 @@
+#pragma once
+
+#include "Constants.h"
+#include "rev/CANSparkMax.h"
+#include "subsystems/BaseSingleAxisSubsystem.h"
+
+class WristSubsystem
+    : public BaseSingleAxisSubsystem<rev::CANSparkMax,
+                                     rev::SparkMaxAbsoluteEncoder,
+                                     units::degree, units::degree_t> {
+   public:
+    WristSubsystem()
+        : BaseSingleAxisSubsystem(m_config, m_wristMotor, m_encoder, &min,
+                                  nullptr, "WRIST", "\033[92;40;4m") {
+        _config = m_config;
+        _controller = m_config.pid;
+        _controller.SetTolerance(10, 10);
+        m_encoder.SetPositionConversionFactor(1);
+    }
+
+    // Wrist has zero offset set in SparkMax
+    void ResetEncoder() override {
+        if (_log)
+            Logging::logToStdOut(_prefix, "RESET POSITION",
+                                 Logging::Level::INFO);
+        // m_encoder.SetZeroOffset(0);
```

Comment:
method currently does nothing, is this correct?

Reply:
Correct since the 0 for the absolute encoder is being set on the SparkMax itself. We only want to log here.